README.md

```diff
@@ -65,19 +68,104 @@ See [these instructions for configuring Tomcat](https://github.com/unidata/tomca
 
     ```bash
     $ docker run \
+        -p 8080:8080 \
         -v /path/to/your/setup.xml:/usr/local/tomcat/content/erddap/setup.xml \
         -v /path/to/your/datasets.xml:/usr/local/tomcat/content/erddap/datasets.xml \
         ... \
         axiom/docker-erddap
     ```
 
-    **Any custom setup.xml needs to specify `<bigParentDirectory>/erddapData/</bigParentDirectory>`**
+    **If you mount setup.xml file make sure to set `<bigParentDirectory>/erddapData/</bigParentDirectory>`**
+
+2. Configure using environmental variables
+
+    You can set environmental variables to configure ERDDAP's `setup.xml` since version 2.14. See the [ERDDAP documentation](https://coastwatch.pfeg.noaa.gov/erddap/download/setup.html#setupEnvironmentVariables) for details. This can be very useful so you don't need to mount a custom `setup.xml` file into your container. If taking this approach you should look into setting the following ERDDAP config options:
+
+    * `ERDDAP_baseURL`
+    * `ERDDAP_baseHttpsUrl`
+    * `ERDDAP_flagKeyKey`
+    * `ERDDAP_emailEverythingTo`
+    * `ERDDAP_emailFromAddress`
+    * `ERDDAP_emailUserName`
+    * `ERDDAP_emailPassword`
+    * `ERDDAP_emailSmtpHost`
+    * `ERDDAP_emailSmtpPort`
+    * `ERDDAP_adminInstitution`
+    * `ERDDAP_adminInstitutionUrl`
+    * `ERDDAP_adminIndividualName`
+    * `ERDDAP_adminPosition`
+    * `ERDDAP_adminPhone`
+    * `ERDDAP_adminAddress`
+    * `ERDDAP_adminCity`
+    * `ERDDAP_adminStateOrProvince`
+    * `ERDDAP_adminPostalCode`
+    * `ERDDAP_adminCountry`
+    * `ERDDAP_adminEmail`
+
+    For example:
+
+    ```bash
+    docker run \
+        -p 8080:8080 \
+        -e ERDDAP_baseURL="http://localhost:8080" \
+        -e ERDDAP_adminEmail="set_via_container_env@example.com" \
+        axiom/docker-erddap
+    ```
+
+3. Configure using a shell script
```

Comment:
I can live with it. Let's hold off on future changes until there are issues, and go with a warning in the docs for now. 

```suggestion
    ```bash
    docker run \
        -p 8080:8080 \
        -e ERDDAP_baseURL="http://localhost:8080" \
        -e ERDDAP_adminEmail="set_via_container_env@example.com" \
        axiom/docker-erddap
    ```

    **Depending on your container environment, it may pass in it's own environment variables relating to your resources. Potentially there could be a collision with the `ERDDAP_*` config variables if any of your resources start with ERDDAP.**

3. Configure using a shell script
```